packages/preview-middleware-client/src/cpe/changes/validator.ts

```diff
@@ -0,0 +1,30 @@
+import ResourceBundle from 'sap/base/i18n/ResourceBundle';
+
+/**
+ * Function to validate if a given value is a valid binding model.
+ *
+ * @param value value to be checked.
+ */
+export function vaildateBindingModel(value: string): void {
+    const bindingValue = value.replace(/{|}/gi, '');
```

Comment:
Plese change as proposed in here https://sonarcloud.io/project/issues?resolved=false&types=CODE_SMELL&pullRequest=1459&id=SAP_open-ux-tools&open=AYvH0V2Hd2OFDSMPrS8P&tab=why to fix the code smell

Reply:
Fixed